src/app.ts

```diff
@@ -16,7 +16,7 @@ const app = new Application();
 // The accesslog middleware shows all requests and responses on the cli.
 app.use(accessLog());
 
-app.use(browser());
+app.use(browser({ title: 'tt-api' }));
```

Comment:
Maybe it could be a full name? This is the human readable title that shows up at the top of the page